docs/guide/extensions.md

```diff
@@ -1,20 +1,20 @@
 # Extensions
 
-The validator's architecture allows to replenish its missing functionality via extensions. A few number already exists 
-and available for usage. Note that they are not official in terms of not being the part of Yiisoft packages.
+The validator's architecture allows for replenishing its missing functionality via extensions. A few already exist
```

Comment:
What's wrong with using "to replenish"?

Reply:
And seems like "allows for" is more appropriate for using with nouns, for example: "allows for things".

So, with this variation, it must be just "allows replenishing".